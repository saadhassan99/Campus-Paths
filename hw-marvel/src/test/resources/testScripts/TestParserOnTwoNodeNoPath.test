# Test the parser on TwoNodeNoPath data to see if the graph is built correctly

# Parse through the data and build the graph
LoadGraph TwoNodeNoPath TwoNodeNoPath.tsv

# List nodes and edges to see if the graph is built correctly
ListNodes TwoNodeNoPath
ListChildren TwoNodeNoPath NodeA
ListChildren TwoNodeNoPath NodeB