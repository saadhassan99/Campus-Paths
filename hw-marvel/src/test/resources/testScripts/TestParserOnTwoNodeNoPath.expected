# Test the parser on TwoNodeNoPath data to see if the graph is built correctly

# Parse through the data and build the graph
loaded graph TwoNodeNoPath

# List nodes and edges to see if the graph is built correctly
TwoNodeNoPath contains: NodeA NodeB
the children of NodeA in TwoNodeNoPath are:
the children of NodeB in TwoNodeNoPath are: 
